services/app-web/src/pages/App/AppRoutes.tsx

```diff
@@ -61,6 +61,11 @@ const UniversalRoutes = (
         <Route path={RoutesRecord.HELP} element={<Help />} />
     </Fragment>
 )
+//Doing this in order to pass the :id for redirecting '/:id' to ':id/edit' for state users viewing rates
+const RateEditRouteRedirect = () => {
```

Comment:
Redirecting should be done on the `RateSummary` page to follow the pattern we set for the submission summary pages.